src/ripple/app/main/Application.cpp

```diff
@@ -1008,7 +1008,7 @@ void ApplicationImp::setup()
         enabledAmendments.append (detail::preEnabledAmendments ());
 
         m_amendmentTable = make_AmendmentTable (
-            weeks(2),
+            weeks{0},  // HH TODO:  Set back to weeks{2} for production deployment
```

Comment:
Two things: 

* Please make all changes that should be removed prior to being deployed in production a separate commit.
* Please make this `minutes{1}`.